# X cell index (starting from 0)
uint8 x
# Y cell index (starting from 0)
uint8 y
